convert2rhel/backup/files.py

```diff
@@ -120,7 +120,8 @@ def restore(self, rollback=True):
             return
 
         try:
-            shutil.move(self._backup_path, self.filepath)
+            shutil.copy2(self._backup_path, self.filepath)
+            os.remove(self._backup_path)
```

Comment:
If there's an exception raised by the `shutil.copy2` (for example the target directory doesn't exist), the `os.remove` is not called.

Reply:
Yes, but that's how we should have it. If something didn't go well with the copy we don't want to remove the backup